### ISSUE TITLE
Figure 12 in Understanding of SC 2.4.11 does not have sufficient contrasts

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag#2227]
[Understanding 2.4.11](https://w3c.github.io/wcag/understanding/focus-appearance-minimum.html): Figure 12 is intended to show which areas of the shadow have sufficient contrast of 3:1 to be considered as focus indicators. Unfortunately, many of the areas are well below 3:1 (e.g. top left and right only 1.5:1). Also, it seems to me that the shadow in Figure 12 does not match the shadow in Figure 11 (and I mean the areas that are not hidden in Figure 12). The shadow in Figure 11 seems to be darker.

**Comment by alastc** [comment]
Hi @JAWS-test, sorry the figures got re-numbered, can you confirm which you mean? Is it this one? https://w3c.github.io/wcag/understanding/focus-appearance-minimum.html#focus-indicator-box-shadow

**Comment by JAWS-test** [comment]
@alastc exactly, now it is the figures 17 and 18

**Comment by mbgower** [comment]
@JAWS-test, first can you please review the text and the caption. I believe the context is provided by the revisions.
Second, I'm not sure how to address this holistically, but especially in this example, it is an image of part of a UI taken to show a process. As such, the image is a compounded pixellation, perhaps 3 times (the original focus indicator is pixellated by the user agent, the screen capture causes some additional distortion, the image with that distortion is then presented in your browser). So, there will almost certainly be some pixels that are not true to the original image.
Even if this was an svg, we'd potentially encounter this problem.
I guess one solution is to add a note about this, so that people aren't using an eyedropper to challenge the exercise, but instead are just absorbing the intent of the demo

**Comment by mbgower** [comment]
If you have the technology to produce a more consistent demonstration, I would welcome you redoing a comparable example!! :)

**Comment by mbgower** [comment]
@JAWS-test I have made the following addition. Please let me know if this addresses your concern:
> NOTE
Some of the examples in this document, especially in the Exceptions section, are screen-captured images of elements. Due to loss of resolution in these images, the actual pixel color may not match the original. As such, they are intended to be used for illustrative purposes, and should not be inspected on a pixel-by-pixel basis for sufficient contrast.

This appears immediately following the gradient examples you cited. This screen shot shows it in context with the preceding Figure (now 19) and the following paragraph. I can be reviewed at https://w3c.github.io/wcag/understanding/focus-appearance-minimum.html
![image](https://user-images.githubusercontent.com/14298245/168394185-7bda237c-c797-4f5e-9f4c-f96707cb8d3a.png)

Please close if you believe this resolves the issue, or provide more comments.
